pkg/api/v1/status/atlasdatabaseuser.go

```diff
@@ -0,0 +1,11 @@
+package status
+
+// +k8s:deepcopy-gen=false
+
+// AtlasDatabaseUserStatusOption is the option that is applied to Atlas Project Status
+type AtlasDatabaseUserStatusOption func(s *AtlasDatabaseUserStatus)
+
+// AtlasDatabaseUserStatus defines the observed state of AtlasProject
+type AtlasDatabaseUserStatus struct {
+	Common `json:",inline"`
```

Comment:
[q] is Common used elsewhere? Seems like we can just put the fields in here directly instead?

Reply:
yes, it's used in two other statuses (for cluster and project)